### PR TITLE
Update hello-planning-guide.md, issue #1887

### PR DESCRIPTION
**Pull request by MaratMussabekov**
fixed wrong terminology, source: [Windows Hello for Business Provisioning](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-how-it-works-provisioning)

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

This is for issue #1887.

**Comment by JohanFreelancer9** [comment]
Hello Joyce. Next issue for me. Why, when I want to click on the + symbol
to do the copy edit, do I get the message "Suggestions cannot be applied
while viewing a subset of changes"?

**Comment by mypil** [comment]
@JohanFreelancer9 - that I'm not sure of.. Looping in @kenwith as he may know.

**Comment by kenwith** [comment]
I haven't experienced that one either. This editing within the PR a new feature of GitHub so it will take some time for us all to work out how it works. When I click the green + symbol I get a textbox where I can type a suggestion. Or I can click the little 'suggest changes' button in the middle at the top and it brings the line of text into the textbox and I can make changes there.

**Comment by mypil** [comment]
@DaniHalfin - can you please assist to check, approve and merge the content for this PR based on issue #1887?

Thank you.

**Comment by DaniHalfin** [comment]
Looks good to me, but want to make sure @mapalko gives it a look.

**Comment by mapalko** [comment]
@mypil and @DaniHalfin, is out-of-box the correct terminology for experiences built into the OS? If it is, then it looks good. I want to make sure, since traditionally we refer to out-of-box experience just for the first time you set up the device. This specific provisioning experience can happen at a later time.

**Comment by DaniHalfin** [comment]
After talking to @mapalko - I agree that out-of-box might be missleading. We can altogether drop that part and just have a link to the source referenced at the top to clarify.
The reference here is to the provisioning experience built into Windows, rather than a custom provisioning experience (that was made through a 3rd party API).

**Comment by DaniHalfin** [comment]
@mypil - see comments above (forgot to tag)

**Comment by mypil** [comment]
@MaratMussabekov - please check and make the necessary changes as recommended by @DaniHalfin. Thank you.

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 1]
```diff
@@ -75,21 +75,21 @@ It’s fundamentally important to understand which deployment model to use for a
...
   
-The key trust type does not require issuing authentication certificates to end users.  Users authenticate using a hardware-bound key created during an in-box provisioning experience, which requires an adequate distribution of Windows Server 2016 domain controllers relative to your existing authentication and the number of users included in your Windows Hello for Business deployment.  Read the [Planning an adequate number of Windows Server 2016 Domain Controllers for Windows Hello for Business deployments](hello-adequate-domain-controllers.md) to learn more.
+The key trust type does not require issuing authentication certificates to end users.  Users authenticate using a hardware-bound key created during an out-of-box provisioning experience, which requires an adequate distribution of Windows Server 2016 domain controllers relative to your existing authentication and the number of users included in your Windows Hello for Business deployment.  Read the [Planning an adequate number of Windows Server 2016 Domain Controllers for Windows Hello for Business deployments](hello-adequate-domain-controllers.md) to learn more.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]: The key trust type does not require issuing authentication certificates to end users.  Users authenticate using a hardware-bound key created during an out-of-box provisioning experience**. This** requires an adequate distribution of Windows Server 2016 domain controllers relative to your existing authentication and the number of users included in your Windows Hello for Business deployment.  Read the [Planning an adequate number of Windows Server 2016 Domain Controllers for Windows Hello for Business deployments](hello-adequate-domain-controllers.md) to learn more.

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 2]
```diff
@@ -75,21 +75,21 @@ It’s fundamentally important to understand which deployment model to use for a
...
 
-The certificate trust type issues authentication certificates to end users.  Users authenticate using a certificate requested using a hardware-bound key created during the in-box provisioning experience.  Unlike key trust, certificate trust does not require Windows Server 2016 domain controllers.  Users can authenticate using their certificate to any Windows Server 2008 R2 or later domain controller.
+The certificate trust type issues authentication certificates to end users.  Users authenticate using a certificate requested using a hardware-bound key created during the out-of-box provisioning experience.  Unlike key trust, certificate trust does not require Windows Server 2016 domain controllers.  Users can authenticate using their certificate to any Windows Server 2008 R2 or later domain controller.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]: Copy edited, no changes.

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 3]
```diff
@@ -75,21 +75,21 @@ It’s fundamentally important to understand which deployment model to use for a
...
 
-The in-box Windows Hello for Business provisioning experience creates a hardware bound asymmetric key pair as their user’s credentials. The private key is protected by the device’s security modules; however, the credential is a user key (not a device key).  The provisioning experience registers the user’s public key with the identity provider.  For cloud only and hybrid deployments, the identity provider is Azure Active Directory.  For on-premises deployments, the identity provider is the on-premises server running Windows Server 2016 Active Directory Federation Services (AD FS) role.
+The out-of-box Windows Hello for Business provisioning experience creates a hardware bound asymmetric key pair as their user’s credentials. The private key is protected by the device’s security modules; however, the credential is a user key (not a device key).  The provisioning experience registers the user’s public key with the identity provider.  For cloud only and hybrid deployments, the identity provider is Azure Active Directory.  For on-premises deployments, the identity provider is the on-premises server running Windows Server 2016 Active Directory Federation Services (AD FS) role.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]: Copy edited, no changes.

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 4]
```diff
@@ -75,21 +75,21 @@ It’s fundamentally important to understand which deployment model to use for a
...
 
 #### Multifactor authentication
 
-The goal of Windows Hello for Business is to move organizations away from passwords by providing them a strong credential that provides easy two-factor authentication.  The in-box provisioning experience accepts the user’s weak credentials (username and password) as the first factor authentication; however, the user must provide a second factor of authentication before Windows provisions a strong credential.  
+The goal of Windows Hello for Business is to move organizations away from passwords by providing them a strong credential that provides easy two-factor authentication.  The out-of-box provisioning experience accepts the user’s weak credentials (username and password) as the first factor authentication; however, the user must provide a second factor of authentication before Windows provisions a strong credential.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]: Copy edited, no changes.

**Comment by MaratMussabekov** [comment]
Dear @DaniHalfin , 
" I agree that out-of-box might be missleading" - which term would be more correct by your opinion? should I left "In-box"? or what?

"just have a link to the source referenced at the top to clarify" - do you mean "Windows Hello for Business Provisioning" article or some other source?

Thank you!

> 
> 
> After talking to @mapalko - I agree that out-of-box might be missleading. We can altogether drop that part and just have a link to the source referenced at the top to clarify.
> The reference here is to the provisioning experience built into Windows, rather than a custom provisioning experience (that was made through a 3rd party API).

Dear @JohanFreelancer9 , thank you very much, I will apply your recommendations after the terminology I should use becomes clear.

**Comment by mypil** [comment]
@DaniHalfin - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR? 

Thank you.

**Comment by DaniHalfin** [comment]
Made a small copy change. As for the larger issue to address - I'd like to see it say "built-in provisioning experience".
The intent here is to differentiate between what comes with Windows vs. 3rd party provisioning experiences. In-box as a term wasn't clear to readers, and out-of-box can be confused with OOBE (out-of-box experience) which is the one you get the first time you start the device. In this case, the provisioning experience can happen at any time (not just first run), hence to avoid confusion we should either avoid terms and just clearly explain the intention, or use something clearer, such as "built-in".

**Comment by DaniHalfin** [comment]
tagging @MaratMussabekov and @mypil because I forgot

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 , @DaniHalfin 
your recommendations were applied
Thank you!

**Comment by DaniHalfin** [comment]
Thanks.

**Pull request merged**